### ISSUE TITLE
Question about different results for the same picture

### ISSUE THREAD

**Issue by xingwen78** [jazzsaxmafia/show_and_tell.tensorflow#9]
Hi

I downloaded the code and it worked very well, nice stuff.
Just one question, when testing a same picture I got different results from time to time.
Is that just because of the weights' float-point rounding ?

Thank you.

**Comment by chingyaoc** [comment]
I have the same problem while running test_tf(). 
The output of VGG is different from time to time.
Any suggestion?

**Comment by kilickaya** [comment]
Same here. I first suspected the problem is due to random initialized parameters of textual and image embeddings, however as they are loaded from the model-72, the only reason for different results are due to different VGG activations.

**Comment by YuDeng** [comment]
I'm wondering if this is because of the dropout layers? Is it still working at testing stage?

**Comment by kilickaya** [comment]
Yes, this is probably due to dropout dropping some random connections at test time, which is connected to fc-7 layer (previous layers do not produce random activations) Is there any way to shut it down at test time?

**Comment by ilovezhenzhen** [comment]
Hi 
 ImportError: This module is deprecated.  Use tf.nn.rnn_cell instead.
i use from tensorflow.python.ops import rnn_cell instead from tensorflow.models.rnn import rnn_cell but
ValueError: setting an array element with a sequence.

**Comment by nqkhai1706** [comment]
Hi,
I think the result is different, that is normal. It pick the top scores result by random, not only 1 result
